libraries/testkit/syncgateway.py

```diff
@@ -23,6 +24,10 @@ def __init__(self, cluster_config, target):
         self._headers = {'Content-Type': 'application/json'}
         self.admin = Admin(self)
 
+        self.cluster_config = cluster_config
+        self.server_port = 8091
+        self.server_scheme = "http"
```

Comment:
These are the default non-ssl values. It is also used in other methods like restart. I thought it is a good idea to initialize the defaults when the object is initialized